.github/workflows/security.yaml

```diff
@@ -42,6 +42,8 @@ jobs:
     runs-on: "buildjet-2vcpu-ubuntu-2204"
     steps:
       - uses: "actions/checkout@v4"
+        with:
+          fetch-depth: 0
```

Comment:
should this be fetch-tags or does this work?

Reply:
>Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set fetch-depth: 0 to fetch all history for all branches and tags. Refer [here](https://docs.github.com/actions/using-workflows/events-that-trigger-workflows) to learn which commit $GITHUB_SHA points to for different events.

We should add a comment to this line

Reply:
>should this be fetch-tags or does this work?

@ecordell I tried that first. For some reason that didn't work. Could be an issue with the GH action, so I went the easy route

Reply:
https://github.com/actions/checkout/issues/1471